src/classes/sentry.py

```diff
@@ -47,12 +48,16 @@ def init_tracing():
         return
 
     # Determine sample rate for exceptions
-    traces_sample_rate = 0.1
-    environment = "production"
-    if "-dev" in info.VERSION:
-        # Dev mode, trace all exceptions
-        traces_sample_rate = 1.0
-        environment = "development"
+    traces_sample_rate = 0.0
+    if info.VERSION == info.ERROR_REPORT_STABLE_VERSION:
+        traces_sample_rate = info.ERROR_REPORT_RATE_STABLE
+        environment = "production"
+    else:
+        traces_sample_rate = info.ERROR_REPORT_RATE_UNSTABLE
+        environment = "unstable"
```

Comment:
If I'm understanding... We will a dial for stable, and unstable? That'll be very useful when we come up to another release

Reply:
Exactly, the website will return `{"error_rate_unstable": 0.15, "openshot_version": "2.5.1", "error_rate_stable": 1.0}` when reporting the current stable version.